app/api/auth/request/route.ts

```diff
@@ -1,59 +1,38 @@
-import { NextRequest, NextResponse } from "next/server"
+import { NextResponse } from 'next/server'
 
-const AUTH_BASE_URL = process.env.HYDRA_PUBLIC_URL + "/oauth2"
+const AUTH_BASE_URL = process.env.HYDRA_PUBLIC_URL + '/oauth2'
 const SEP_REDIRECT_URI = process.env.NEXT_PUBLIC_SEP_REDIRECT_URI ?? ''
```

Comment:
@callumstew I've moved the url request to the API so that it can get the env variables during runtime. It doesn't seem to be pulling the vars otherwise.